config/environments/development.rb

```diff
@@ -59,4 +59,5 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.hosts.clear
```

Comment:
cái này có tác dụng gì em

Reply:
dạ a , nó có tác dụng xóa toàn bộ danh sách trắng, cho phép thông qua các yêu cầu cho tất cả tên máy chủ ạ